cpp/custrings/text/edit_distance.cu

```diff
@@ -47,12 +47,16 @@ struct editdistance_levenshtein_algorithm
     editdistance_levenshtein_algorithm( custring_view** strings, custring_view** tgts, short* buffer, size_t* offsets, unsigned int* results )
     : d_strings(strings), d_tgt(0), d_tgts(tgts), d_buffer(buffer), d_offsets(offsets), d_results(results) {}
 
+    // pairwise strings
+    editdistance_levenshtein_algorithm( custring_view** strings, short* buffer, size_t* offsets, unsigned int* results )
```

Comment:
Please wrap long lines (longer than 80 columns). My general rule: if you have to scroll right to review it on github, it's too wide.

Reply:
Thanks, I'll follow this rule from now on!